tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -17,7 +17,8 @@
 from tango import CmdArgType, AttrDataFormat
 
 from tango_simlib.utilities.base_parser import Parser
-from tango_simlib.utilities.helper_module import json_load_byteified
+#from tango_simlib.utilities.helper_module import json_load_byteified
+from helper_module import json_load_byteified
```

Comment:
- [ ] Please fix the import statement. Line 20 is the correct way to import that function.